lib/final_destination.rb

```diff
@@ -203,12 +205,21 @@ def resolve
     middlewares = Excon.defaults[:middlewares]
     middlewares << Excon::Middleware::Decompress if @http_verb == :get
 
+    time = Time.now
+    body = +""
```

Comment:
What is the `+""` doing here? Maybe just a ruby feature I don't know :)

Reply:
Maybe name this `response_body` too so we are clear, and `time` into `request_start_time`. So then we will have `Time.now - request_start_time > MAX_REQUEST_TIME_SECONDS` which I think is clearer

Reply:
+"" is creating unfrozen string, otherwise it is throwing error when modify